storio-common/src/main/java/com/pushtorefresh/storio/operations/PreparedOperation.java

```diff
@@ -62,4 +62,7 @@
     @NonNull
     @CheckResult
     Single<Result> asRxSingle();
+
+    @NonNull
+    Object getData();
```

Comment:
With interface it would be impossible to use third side classes. 
We can add generic param to all prepared operation, if you think that it worth doing.